kinetic/distribution.yaml

```diff
@@ -4758,6 +4758,21 @@ repositories:
       url: https://github.com/davetcoleman/moveit_visual_tools.git
       version: kinetic-devel
     status: developed
+  movidius_ncs:
+    doc:
+      type: git
+      url: https://github.com/intel/ros_intel_movidius_ncs.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/intel/ros_intel_movidius_ncs.git
```

Comment:
This is failing validation as this is not a gbp release repo generated by bloom. If you just want indexing please only add the doc and source entries, or from the current state remove the release entry. The release entry should be autofilled by bloom running through the release process: http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease

Reply:
@tfoote, thanks for your detailed comments. I will remove release entry first and submit again.